.golangci.yml

```diff
@@ -7,5 +7,11 @@ linters:
     - exportloopref
     - gofumpt
     - misspell
+    - nakedret
     - revive
     - prealloc
+
+linters-settings:
+  nakedret:
+    # Ban the use of naked returns because they reduce code readability.
```

Comment:
note: this is subjective but I expect the linter will reduce the # of nits / comments on PRs which is a net positive